infrastructure/main.tf

```diff
@@ -98,42 +98,4 @@ module "db-user-profile" {
   postgresql_user = "dbuserprofile"
   database_name = "dbuserprofile"
   common_tags = "${var.common_tags}"
-}
-
-module "rd-user-profile-api" {
-  source = "git@github.com:hmcts/cnp-module-webapp?ref=master"
-  product = "${var.product}-${var.component}"
-  location = "${var.location}"
-  env = "${var.env}"
-  ilbIp = "${var.ilbIp}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  subscription = "${var.subscription}"
-  capacity = "${var.capacity}"
-  instance_size = "${var.instance_size}"
-  common_tags = "${merge(var.common_tags, map("lastUpdated", "${timestamp()}"))}"
-  appinsights_instrumentation_key = "${var.appinsights_instrumentation_key}"
-  asp_name = "${local.app_service_plan}"
-  asp_rg = "${local.app_service_plan}"
-  enable_ase = "${var.enable_ase}"
-
-  app_settings = {
-    LOGBACK_REQUIRE_ALERT_LEVEL = false
-    LOGBACK_REQUIRE_ERROR_CODE = false
-
-    POSTGRES_HOST = "${module.db-user-profile.host_name}"
-    POSTGRES_PORT = "${module.db-user-profile.postgresql_listen_port}"
-    POSTGRES_DATABASE = "${module.db-user-profile.postgresql_database}"
-    POSTGRES_USER = "${module.db-user-profile.user_name}"
-    POSTGRES_USERNAME = "${module.db-user-profile.user_name}"
-    POSTGRES_PASSWORD = "${module.db-user-profile.postgresql_password}"
-    POSTGRES_CONNECTION_OPTIONS = "?"
-
-    IDAM_URL = "${data.azurerm_key_vault_secret.idam_url.value}"
-    S2S_URL = "${local.s2s_url}"
-
-    ROOT_LOGGING_LEVEL = "${var.root_logging_level}"
-    LOG_LEVEL_SPRING_WEB = "${var.log_level_spring_web}"
```

Comment:
is it safe to remove the app module? Have we analyzed if there is any impact in production due to removing the app service?

Reply:
Have retained the webapp as advised.